### ISSUE TITLE
Play Button not working

### ISSUE THREAD

**Issue by amrashraf1992** [cmonapp/android-facebook-player#1]

**Comment by cmoijulien** [comment]
Yes play function seems to be not working. If you click on play icon on the video is working. 
I think it's problem with API Embedded Video Player the play function is called but nothing happens.
https://developers.facebook.com/docs/plugins/embedded-video-player/api/
